chore: clean up the repository (#121)
asyncapi/extensions-catalog: extensions/twitter/README.md

# Twitter extension
This document defines how to use `twitter` extension in AsyncAPI documents.

## Overview 
This extension allows you to provide the Twitter username of the account representing the team/company of the API.

## Extension Definition

### Type: String

Name of the Twitter username.

> ### Extension Location 

Reviewer: ```suggestion
## Extension Location 
```

actually it should be h2 😄

# Twitter extension
This document defines how to use `twitter` extension in AsyncAPI documents.

## Overview 

> This extension allows you to place the twitter account in charge of the team/company of the API.

Reviewer: ```suggestion
This extension allows you to provide the Twitter username of the account representing the team/company of the API.
```

## Extension Definition

> Extensions are the mechanism AsyncAPI has to allow you use custom or protocol-specific features. Extensions follow a separate release cycle and everyone can create their own. This repository is meant to contain a list of official and community supported extensions.

Reviewer: it is not twitter related info, please remove

it is ok if this is empty, idea is that it is `h2` as it just introduces the following `h3`

Author: So we don't need to have extension definition here?

Reviewer: you mean what extensions are in genera? no

this header says that below you have extension definition, which is its type and supported location

Author: ok got it , AI added the extension location to h3

> Both, JSON or YAML, are supported formats. Please, take into account that only the subset of YAML that can be translated to JSON is allowed. 
### Type: String

> Name of the Twitter profile.

Reviewer: ```suggestion
Name of the Twitter username.
```

just to follow how twitter calls it in their UI

## Extension Location 

This extension can be used in the following locations:

> - [Info Object](https://www.asyncapi.com/docs/reference/specification/v2.5.0#infoObject)

Reviewer: ```suggestion
- [Info Object](https://www.asyncapi.com/docs/reference/specification/v2.6.0#infoObject)
```

## Example

```yaml

> asyncapi: '2.5.0'

Reviewer: ```suggestion
asyncapi: '2.6.0'
```

# Twitter extension
This document defines how to use `twitter` extension in AsyncAPI documents.

> ## Twitter Object

Reviewer: please have a look at "what's an extension"

Author: I didn't get it ,can you please elaborate a little.

Reviewer: oh sorry, my comment had more content 🤔 

please look at https://github.com/asyncapi/extensions-catalog/pull/112#pullrequestreview-1182610285 as this document needs restructure

Author: thanks this will help a lot 😀

Reviewer: I think only this one is missing

Author: @derberg just to clarify I need to add the elaborate what is an extension?
